packages/frameworks/src/frameworks.ts

```diff
@@ -1027,6 +1027,45 @@ export const frameworks = [
     },
     getOutputDirName: async () => 'public',
   },
+  {
+    name: 'SvelteKit 2',
+    slug: 'sveltekit-2',
+    demo: 'https://sveltekit-template.vercel.app',
+    logo: 'https://api-frameworks.vercel.sh/framework-logos/svelte.svg',
+    screenshot:
+      'https://assets.vercel.com/image/upload/v1647366075/front/import/sveltekit.png',
+    tagline:
+      'SvelteKit is a framework for building web applications of all sizes.',
+    description: 'A SvelteKit app optimized Edge-first.',
+    website: 'https://kit.svelte.dev',
+    envPrefix: 'VITE_',
```

Comment:
I thought you said the newer version of SveltKit no longer needs a prefix to inject env vars in the client bundle?